info/v1/machine.go

```diff
@@ -112,6 +112,22 @@ type NetInfo struct {
 	Mtu int64 `json:"mtu"`
 }
 
+type CloudProvider string
+
+const (
+	GCE              CloudProvider = "GCE"
+	AWS                            = "AWS"
+	BAREMETAL                      = "BAREMETAL"
```

Comment:
nit: Go style is camelcase, this is fine for the above, but for the others it should be Baremetal and UnknownProvider